README.md

```diff
@@ -55,72 +30,74 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Capstone Project #1 <a name="about-project"></a>
+
+<div align="center">
+
+  <img src="web_mockup.png" alt="logo" width="450"  height="auto" />
+  <br/>
+
+  <h3><b>Capstone Project n1 | README</b></h3>
 
-> Describe your project in 1 or 2 sentences.
+</div>
+
+**Conference web** is a project for Microverse bootcamp to implement all the learnd about Mobile first and responsive design...
 
-**[your_project__name]** is a...
+It is a conference web on stoic phylosophy.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
-  <summary>Client</summary>
+  <summary>HTML</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="#">HTML</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>CSS</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="#">CSS</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+<summary>Github</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#">Github</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Mobile version**
+- **Headline**
+- **Introduction to the portfolio**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+[![Loom video demo](https://cdn.loom.com/sessions/thumbnails/1e1bbfa9d468438091e43055e7d138dc-with-play.gif)](https://www.loom.com/share/1e1bbfa9d468438091e43055e7d138dc)
```

Comment:
- **[OPTIONAL]** Sorry this wasn't mentioned before I see that the video presentation looks amazing, and you explained the purpose of the project very well, to make it more interesting I highly suggest highlighting a piece of the code that you feel proud of making. this will convey that you constantly improving yourself and demonstrate that you making progress 💪🏼 📈 .